src/mozanalysis/experiment.py

```diff
@@ -771,7 +774,7 @@ def _build_enrollments_query_fenix_baseline(
                 '{experiment_slug}'
             ).branch IS NOT NULL
             AND b.sample_id < {sample_size}
-        GROUP BY client_id, branch
+        GROUP BY b.client_info.client_id, branch
```

Comment:
Do we run `fenix_baseline` experiments any more? Was this change related to the PR or is this fixing an old issue?

Reply:
Not sure on the first part, but I was just updating everything where I either saw an issue or wanted to make things consistent. I think this one in particular was an issue because I renamed the alias client_id to analysis_id and so I could no longer group on client_id. However, I'm pretty sure the way SQL is evaluated means you shouldn't be able to use aliases in GROUP BY anyway (bigquery might do some stuff here to make it work), so instead of changing to analysis_id I just went with the full original field.